src/main/java/org/desarrolladorslp/technovation/controller/BatchController.java

```diff
@@ -75,6 +69,13 @@
         return new ResponseEntity<>(batches.stream().map(this::convertToDTO).collect(Collectors.toList()), HttpStatus.OK);
     }
 
+    @Secured({"ROLE_ADMINISTRATOR"})
+    @DeleteMapping("/{batchId}")
+    public ResponseEntity<Batch> deleteBatch(@PathVariable String batchId){
```

Comment:
It says you should be returning a Batch ... in this case you can simply replace `ResponseEntity<Batch>` with `void` and remove the `return` statement.